app/src/main/java/com/sun/efood/utils/FragmentUtil.kt

```diff
@@ -0,0 +1,50 @@
+package com.sun.efood.utils
+
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentManager
+import androidx.fragment.app.FragmentTransaction
+
+object FragmentUtil {
+
+    fun addFragment(
+        fragmentManager: FragmentManager,
+        layoutId: Int,
+        fragment: Fragment,
+        tag: String? = null
+    ) {
+        fragmentManager.beginTransaction()
+            .apply {
+                setTransition(FragmentTransaction.TRANSIT_FRAGMENT_OPEN)
+                add(layoutId, fragment, tag)
+                addToBackStack(tag)
+                commit()
+            }
```

Comment:
anh nhớ k nhầm thì fragment manager đã viết dưới dạng builder pattern rồi mà nhỉ, nên không cần đặt trong scope function apply nữa 
```suggestion
        fragmentManager.beginTransaction()
            .setTransition(FragmentTransaction.TRANSIT_FRAGMENT_OPEN)
            .add(layoutId, fragment, tag)
                addToBackStack(tag)
                commit()
           
```